kansha/board/boardsmanager.py

```diff
@@ -59,15 +60,23 @@ def _get_board(self, data_board):
         return self._services(Board, data_board.id, self.app_title, self.app_banner, self.theme,
                               self.card_extensions, self.search_engine)
 
+    def create_template_empty(self):
```

Comment:
Note: this method will have to be removed when the importation of board will be available